package.json

```
@@ -40,7 +40,8 @@
     "stage-release": "ng run tools-release:build-stage-release && node dist/libs/tools/stage-release/main.js",
     "publish-release": "ng run tools-release:build-publish-release && node dist/libs/tools/publish-release/main.js",
     "barista:prerender": "ng run barista-design-system:prerender --routesFile dist/barista-data/routes.txt",
-    "bazel:format": "npm run bazel run //:buildifier"
+    "bazel:format": "bazel build //:buildifier",
+    "bazel:build": "bazel build //libs/workspace/src/builders/barista-build --remote_upload_local_results=true --google_credentials=tools/bazel_build_cache/cache-write-key.json"
```

should we use a bazel query here for detecting all build targets?
```suggestion
    "bazel:build": "bazel build //... --remote_upload_local_results=true --google_credentials=tools/bazel_build_cache/cache-write-key.json"
```

---

As the development of build targets is still very much ongoing and rather unstable I'd like to have this on a fixed target to not break PR builds of contributors or introduce any side-effects.

I would change to querying once we have a stable set of build targets that can reliably be run on the CI

---

Ok for now but I want to build always all bazel targets so that we cannot check in broken code as it would effect others as well not only ci.

---

Yes I totally agree, but with this PR I think it is just too early.